### DISCUSSION TITLE
Anyone with smallish battery? Like 1kwh or similar?

### DISCUSSION THREAD

**Discussion by greymda** [hoylabs/OpenDTU-OnBattery#941] [Show and tell]
Looking for some advice as i am pretty new to all this but really want to build something by myself.
So, if you are using a smallish battery please chime in as i have some questions.

For example- i get that i need a fuse between Victron MPPT and the battery.
 I plan to use 6mm wires - is this okay?
If yes, do i need a 20A fuse, or 25, or 30?

Apologies if this sounds silly, but bear with me please.

Edit: forgot to add, all connected to a Hoymiles HM-800.

**Comment by spcqike** [comment]
Fuse size does not depend on battery capacity. It depends on battery discharge current and wire diameter. 

For 6mm^2 you could use any of these fuses. But only if your battery can safely handle that much discharge current.

**Comment by greymda** [comment]
it’s 16 of these guys in series https://www.evlithium.com/hot-lithium-battery/40135-20ah-cylindrical-lifepo4-cell.html

Standard Charging Current 10000mA
Standard Discharge Current 10000mA
Max Continuous Discharge Current 60000mA

**Reply by spcqike** [reply]
> High rate discharge: Capable of maximum 50A 

so it has different ratings given in the same site. 

As it’s a 20Ah cell, I wouldn’t go much above 20A. So I would probably look for a fuse between 20-30A. Which gives you something between 1-1.5kW of discharge power. 

Edit:
What BMS are you using? The BMS could limit your max current, too.

**Reply by greymda** [reply]
@spcqike  my total capacity is at around 1kWh. 
the BMS is  JK-BD4A20S4P https://www.jkbms.com/product/jk-bd4a20s4p/

**Reply by greymda** [reply]
forgot to add, all connected to a Hoymiles HM-800.

**Comment by DonJohnLong** [comment]
How much solar imput do you have? These 20A cell's seem a little bit small to me.

**Reply by greymda** [reply]
@DonJohnLong i have 2 x 440w and don’t have any options to add more now. and a Hoymiles HM-800.

**Reply by spcqike** [reply]
You shouldn’t charge it with 40A anyway if your cable or battery only can handle 30A. And for the cells, I’m not to sure which values to trust. As also for bigger and quality battery’s you have ratings like 0.8C or 1C, which would be 16A-20A here.

**Reply by DonJohnLong** [reply]
These cells are used in EVs, they can handle higher charge/discharge currents than our usual prismatic cells. 
However, in my opinion you should have a battery which is at least double your solar input. 

In my case, I have 2.4kw peak solar input and a 100ah 48v battery (5.3kwh) on a sunny day the battery is full at 12:00. On rainy days, it is not always possible to charge to 100% soc.

Also, you should cycle your cells between 95-10% to prolong their lifespan which lowers your capacity even more.

I recommend you to order different cells or iv you already ordered them get an additional 16 and make a 16s2p battery.

I assume you have the victron 100/20 MPPT? Remember to wire you solar panels in series, otherwise you will not be able to charge a 48v battery.

**Reply by DonJohnLong** [reply]
6mm2 cables should be plenty, a fuse between the victron and your busbars is not necessary in my opinion (will not trip anyway), however you should have a breaker for your solar input and for your battery output.

Something like this:
https://a.aliexpress.com/_Eu9RZ51

**Reply by greymda** [reply]
@DonJohnLong oh, that’s interesting. so skip
the fuse and put 2 breakers? what amperage?

I have a Victron 150/35. 

By the way, the idea behind mu setup
as i am newbie i just wanted to “recreate”
the Zendure AB1000 setup.

**Reply by greymda** [reply]
@spcqike here’s the official datasheet https://www.evebatteryusa.com/_files/ugd/b4afc3_5f83846234374b7fa995bb3a913d3447.pdf

**Reply by spcqike** [reply]
As it says on page 9: 0.5C standard charge and discharge current. So 10A. However you can charge with 1C up to 80% SoC and you can discharge up to 3C but only for 30s pulse. 

So I would stay within 10A I guess. Max 20A.

**Reply by greymda** [reply]
@spcqike another stupid question: where inset this limit? in the victron mppt, or in opendtu-onbattery?
